eth-providers/src/base-provider.ts

```diff
@@ -741,8 +743,38 @@ export abstract class BaseProvider extends AbstractProvider {
 
   // Queries
   getTransaction = (transactionHash: string): Promise<TransactionResponse> => throwNotImplemented('getTransaction');
-  getTransactionReceipt = (transactionHash: string): Promise<TransactionReceipt> =>
-    throwNotImplemented('getTransactionReceipt');
+  getTransactionReceipt = async (transactionHash: string): Promise<TransactionReceipt> => {
+    const tx = await getTxReceiptByHash(transactionHash);
+
+    if (!tx) {
+      return logger.throwError(`transaction hash not found`, Logger.errors.UNKNOWN_ERROR, { transactionHash });
+    }
+
+    // TODO: correct values of these?
+    const confirmations = 0;
+    const effectiveGasPrice = BIGNUMBER_ONE;
+    const byzantium: false = false;
+    const defaultAddress = '0x';
+
+    return {
+      to: tx.to || defaultAddress,
+      from: tx.from,
+      contractAddress: tx.contractAddress || defaultAddress,
```

Comment:
according to https://infura.io/docs/ethereum/json-rpc/eth-getTransactionReceipt it should be null if no contractAddress

> the contract address created, if the transaction was a contract creation, otherwise - null.